docs/data/material/customization/overriding-component-structure/overriding-component-structure.md

```diff
@@ -0,0 +1,108 @@
+# Overriding component structure
+
+<p class="description">Learn how to override the default DOM structure of Material UI components.</p>
+
+Material UI components are designed to suit the widest possible range of use cases, but you may occasionally need to change how a component's structure is rendered in the DOM.
+
+To understand how to do this, it helps to have an accurate mental model of the components.
```

Comment:
If you take my suggestion for rearranging the "Context" section, then I would update this line to lead in to the next section more smoothly:

```suggestion
To understand how to do this, it helps to know a bit about how the API design has evolved over time, and to have an accurate mental model of the components themselves.
```